test/partition.jl

```diff
@@ -0,0 +1,95 @@
+# Partition tests
```

Comment:
I trust that these pass. Let's set up CI/CD ASAP!

Reply:
will do